### DISCUSSION TITLE
Modeling of wind turbines with different rotation directions

### DISCUSSION THREAD

**Discussion by robytian** [OpenFAST/openfast#2520] [General]
Dear all,

I tried to create two turbines with different rotational directions using the Aerodyn_driver module, one by adjusting the initial yaw of the turbine to -180 degrees, and the other method by setting the RPM to negative, and both methods gave negative power coefficients as well as a negative angle of attack, so I tried to modify the airfoil coefficients file again to avoid this, and I got the following error. I would like to know what is causing this error and what is a better way for me to achieve a reverse rotation of the rotor and get the correct values?
![image](https://github.com/user-attachments/assets/87c921a3-4459-4c01-be85-3bdf9a60d1f1)

**Comment by jjonkman** [comment]
Dear @robytian,

Can you clarify what you changed in the airfoil coeffiicents to trigger this error?

That said, AeroDyn does not currently support the modeling of wind turbine rotors that spin counter clockwise when looking downwind.  See the following open issue for more information: https://github.com/OpenFAST/openfast/issues/1194.

Best regards,

**Reply by robytian** [reply]
I've solved the problem with this error, it was an oversight on my part that caused me to give two values corresponding to the same angle of attack.

**Reply by robytian** [reply]
Also I was wondering if I copied the coefficients corresponding to angles of attack from 0 to 180 to give 0 to -180 degrees (the lift resistance coefficients are centre symmetric), would that not make the results closer. Because the calculated angle of attack is negative and the range is very close to that of the positive angle of attack.

**Reply by jjonkman** [reply]
Dear @robytian,

Sorry, but I have not previously thought about the approach you are proposing and its implication on the results.  Please share what you learn.

Best regards,
